apps/build/components/Login/config.ts

```diff
@@ -0,0 +1,36 @@
+import { createConfig, http } from 'wagmi';
+import { mainnet } from 'wagmi/chains';
+import { coinbaseWallet, injected, safe, walletConnect } from 'wagmi/connectors';
+import type { Chain } from 'wagmi/chains';
+import { RPC_URLS } from 'libs/util-constants/src';
+
+export const projectId = process.env.NEXT_PUBLIC_WALLET_PROJECT_ID;
+
+export const SUPPORTED_CHAINS: [Chain, ...Chain[]] = [mainnet];
+
+const walletConnectMetadata = {
+  name: 'OLAS Build',
+  description: 'OLAS Build Web3 Modal',
+  url: 'https://build.olas.network', // origin must match your domain & subdomain
+  icons: ['https://avatars.githubusercontent.com/u/37784886'],
+};
+
+export const wagmiConfig = createConfig({
+  chains: SUPPORTED_CHAINS,
+  connectors: [
+    injected(),
+    walletConnect({
+      projectId: process.env.NEXT_PUBLIC_WALLET_PROJECT_ID || '',
+      metadata: walletConnectMetadata,
+      showQrModal: false,
+    }),
+    safe(),
```

Comment:
please let's remove it, it doesn't work correctly if trying to connect with safe